### DISCUSSION TITLE
Uncaught ReferenceError: require is not defined

### DISCUSSION THREAD

**Discussion by mojomaayan** [babel/babel#14956] [Q&A]
### 💻

- [ ] Would you like to work on a fix?

### How are you using Babel?

Other (Next.js, Gatsby, vue-cli, ...)

### Input code

In the [URL](https://padips.netlify.app/courses/2022/),  the code, `<script src="[/app.js](https://padips.netlify.app/app.js)"></script>` throws the following error. I have attached the [public GH repo  ](https://github.com/mojomaayan/p1).

`Uncaught ReferenceError: require is not defined`

### Current and expected behavior

I expect babel not to throw bugs with respect to the require commands.

### Environment

Github repo: https://github.com/mojomaayan/p1 

I am using React as part of Hugo

**Comment by nicolo-ribaudo** [comment]
By default, Babel compiles your code to CommonJS (targeting Node.js). If you want to run it in the browser, you will also need a bundler (such as webpack, rollup, or parcel) to "put all the files together" and give you a single file that you can then run in your browser.
